README.md

```diff
@@ -91,7 +118,7 @@ options is an object or string with the following defaults:
 
```

Comment:
In the code block just above this, can you re-format as follows? (i.e. use the back tick notation "`" and qualify as Javascript) 

```javascript
{
   flags: 'r',
   encoding: null,
   fd: null,
   mode: 0x666,
   autoClose: true
 }
```